nemo_text_processing/text_normalization/hy/taggers/decimal.py

```diff
@@ -82,11 +82,11 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
         graph_integer = pynutil.insert('integer_part: "') + integers + pynutil.insert('"')
         final_graph_wo_sign = graph_integer + delete_separator + insert_space + graph_fractional
 
-        final_graph_wo_negative = final_graph_wo_sign | get_quantity(final_graph_wo_sign, integers).optimize()
-        self.final_graph_wo_negative = final_graph_wo_negative
+        final_graph_wo_negative = final_graph_wo_sign | get_quantity(final_graph_wo_sign, integers)
+        self.final_graph_wo_negative = final_graph_wo_negative.optimize()
 
         final_graph = optional_graph_negative + final_graph_wo_negative
 
         final_graph = self.add_tokens(final_graph)
-        self.final_graph = final_graph
-        self.fst = final_graph.optimize()
+        self.final_graph = final_graph.optimize()
```

Comment:
? These are the same graph

Reply:
Yes, but I need final_graph as it is used in money class.

Reply:
Then just reuse `decimal.fst`. Most pynini ops are constructive, so you're not going to mutate the attribute.

Reply:
Done. Thank you!